builder/proxmox/step_start_vm.go

```diff
@@ -104,7 +104,7 @@ func generateProxmoxNetworkAdapters(nics []nicConfig) proxmox.QemuDevices {
 		setDeviceParamIfDefined(devs[idx], "macaddr", nics[idx].MACAddress)
 		setDeviceParamIfDefined(devs[idx], "bridge", nics[idx].Bridge)
 		setDeviceParamIfDefined(devs[idx], "tag", nics[idx].VLANTag)
-		devs[idx]["firewall"] = *nics[idx].Firewall
+		devs[idx]["firewall"] = nics[idx].Firewall
```

Comment:
Maybe we can keep the same pattern as above here, since it should accomplish the same thing?
```suggestion
		setDeviceParamIfDefined(devs[idx], "firewall", nics[idx].Firewall)
```

Reply:
I'd have to add a new function to convert bool to string, because `setDeviceParamIfDefined` wants a string.

Or maybe a `setDeviceFlag(b  bool)` ... ?

Reply:
Ah, right. Didn't think that through enough. Unless there's other things that would use a `setDeviceFlag`, I think opting for the simple solution of `strconv.FormatBool` would be enough.

Reply:
I suspect proxmox api will want a "1" instead of "true", but let me double check that first

Reply:
Seems like it does not care 👍